float64 angle3rd
